infrastructure/local/docker-compose-full.yml

```diff
@@ -0,0 +1,107 @@
+services:
+  postgres:
+    image: postgres:14-alpine
+    hostname: postgres
+    environment:
+      - PGPORT=5432
+      - PGUSER=postgres
+      - POSTGRES_HOST_AUTH_METHOD=trust
+      - POSTGRES_USER=postgres
+    ports:
+      - "5432:5432"
```

Comment:
please remove port forwarding, this is a common issue with the issuer node setup that users have another postgres already running. Instead we can "expose" ports to other containers in the project: https://docs.docker.com/reference/compose-file/services/#expose